Add option to set size for which parserPopUp prints an inline string.

somiaj opened the pull request:
In hardcopy output, allow the size (number of characters) that an inline string [A/B/C] vs a bullet list is printed to be configurable via a new option `inlineSize`. Note, this slightly changes the default behavior, if the total characters is `inlineSize = 25`, then the list is now inline instead of a bullet list.

somiaj commented:
I have various problems that use drop down menus to fill in the blank in sentences. I do like these being displayed inline in hardcopy rather than a bullet list, hence adding this feature to configure the size this happens, as a few of my cases where just barely over the break point.

I thought about maybe using this for a longer list of choices to keep them in line, but ran into another issue where a single line `[A/B/C]` can get a little long, and was thinking of allowing configuring the separator, `/`, so I can put a space on each end, and the list `[A / B / C]` can wrap at more natural breaks. I do know I could use `showStatic => 0` in those cases, but I was just seeing what others would think about allowing configuration of the inline separator as well.

Alex-Jordan commented:
This seems fine, although I think that before you have to be strictly less than 25 characters long to get it printed inline, but  here its inline when 25 or less. I didn't test, just reading the code. I may be missing a detail.

drgrice1 commented:
@Alex-Jordan is correct.  The previous condition was `length(join('', @list)) < 25 && !grep(/(\/|\[|\])/, @list)`.  So when that was negated to work with `unless` and the new `inlineSize` option added it should have become `length(join('', @list)) >= $self->{inlineSize} || grep(/(\/|\[|\])/, @list)`.

somiaj commented:
I was aware that I changed the behavior, and even mentioned it in the PR description (as I thought is simplified the logic and the description a bit). I have changed it so now the behavior is the same.

drgrice1 reviewed (approved):
I have no problem with this change, and don't see any reason not to allow it.

pstaabp reviewed (approved):
Looks good.
